dynawo/3rdParty/sundials/patch/MSVC/sundials.patch

```diff
@@ -14,22 +15,10 @@ diff '--exclude=.vscode' -urpN sundials-5.3.0/CMakeLists.txt sundials-5.3.0_new/
  # Project SUNDIALS (initially only C supported)
```

Comment:
if(${CMAKE_VERSION} VERSION_GREATER "3.15.0") is also useless

Reply:
why ? Because of the policy or because of cmake version ?

Reply:
same the policy

Reply:
OK